library/Network/CloudSeeder/Main.hs

```diff
@@ -236,8 +243,16 @@ getParameters provisionType paramSources allParamSpecs dependencies env appName
 
     outputs :: m (S.Set (T.Text, ParameterValue))
     outputs = do
-      maybeOutputs <- mapM (\stackName -> (stackName,) <$> getStackOutputs (mkFullStackName env appName stackName)) dependencies
-      let outputsOrFailure = runErrors $ traverse (extractResult (flip const)) maybeOutputs
+      let localDependencies = dependencies ^.. folded.filtered (\s -> not (s ^. globalStack))
+          localDependencyNames = localDependencies ^.. each.name
+      maybeLocalOutputs <- mapM (\stackName -> (stackName,) <$> getStackOutputs (mkFullStackName env appName stackName)) localDependencyNames
+
+      let globalDependencies = dependencies ^.. folded.filtered (^. globalStack)
+          globalDependencyNames = globalDependencies ^.. each.name
+      maybeGlobalOutputs <- mapM (\stackName -> (stackName,) <$> getStackOutputs (mkFullStackName "global" appName stackName)) globalDependencyNames
```

Comment:
The code to fetch local/global dependencies seems very similar. Could you possibly extract this into a function in a `where` block and eliminate the duplication?